src/database/Database.js

```diff
@@ -0,0 +1,26 @@
+/*
+  Klasa Database udostępnia statyczną metodę connect.
+*/
+
+const mongoose = require("mongoose");
+
+require("dotenv").config();
+
+class Database {
+  static connect() {
+    mongoose.connect(process.env.DATABASE_URI, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    });
+
+    mongoose.connection.on("error", (err) => {
+      console.log(err);
+    });
+
+    mongoose.connection.once("open", function () {
```

Comment:
to może być funkcja strzałkowa (a nawet powinna żeby była spójność stylu z tym co wyżej)